integration-tests/tests/import-cohort-data.js

```diff
@@ -31,7 +33,11 @@ const testImportCohortDataFn = _.flow(
 
   const frame = await findIframe(page);
   await click(frame, svgText({ textContains: 'Has WGS Low' }));
+  // Wait for UI to rerender after filtering - TODO replace delay with a specific trigger to await
+  await delay(Millis.ofSeconds(10));
```

Comment:
Is the problem here that adding a timeout to the `click()` doesn't work because the button is visible before the filtering is rendered? I seem to recall there's a way to wait for the next render cycle to complete, I wonder if that would work here?

Reply:
Ohh, maybe! @pshapiro4broad would it be all right with you if I stepped back and let Jade team iterate further on this? My priority was getting this passing reliably.

Reply:
A wait of 10 seconds seems excessive. But I agree with letting the Jade team take over further improvement of this.